tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -2480,6 +2521,11 @@ def graFinalOffsets(self, kernel, tP):
     graIdx = 0
     swapPerpPara = (((tP["isA"] or tP["isB"]) and kernel["DirectToVgpr%s"%tc]) and (not tP["tlu"]) and tP["nrp"] > 1)
 
+    if tP["isSwizzled"]:
+      tP["swizzledBlockSize"] = self.sgprPool.checkOut(1)
+      module.add(SMovB32(dst=sgpr(tP["swizzledBlockSize"]), src=hex(16*32), comment="SWZ: swizzled block = MI_M(%u) * MI_K(%u) * 2" %(16, 16)))
+
```

Comment:
Move a value SwizzledStride (MI_M * MI_K * 2 = 16 * 32) to a SGPR and then pass to the macro GLOBAL_OFFSET_A_SWIZZLED as I-Stride 
I use a tempSgpr since it would be used only once here.